### ISSUE TITLE
NAv-GAC-facturación P1D con pérdidas

### ISSUE THREAD

**Issue by bpesquero** [PKFEric/webProxima#1048]
TEngo dos contratos en los que está habiendo problemas con la generación de la factura de compra. Se trata de los contratos
COM/00906 y COM/03248

El log es el siguiente

COM/00906 | ES0021000002351125NQ |   | P1D | Error, Las medidas P1D 854,  de la factura FV/2021000722 para el contrato COM/00906 no se corresponden con las medidas de cierre 351
-- | -- | -- | -- | --
COM/03248 | ES0021000013507082FF |   | P1D | Error, Las medidas P1D 6.025,  de la factura FV/2021001196 para el contrato COM/03248 no se corresponden con las medidas de cierre 5.522

REvisando todo, veo que en ambos casos se trata de contratos con pérdidas y por lo que veo hay un error por nuestra parte en el cálculo de pérdidas y es por eso uqe no coinciden las meddias del P1D con las del cierre (que incluye las pérdidas calculadas por nosotros de forma errónea). Sin embargo en ambos casos coincide el P1D con las medidas facturadas en el peaje.

**Comment by JMunoz17** [comment]
Donde tenéis los ficheros Belén?

**Comment by bpesquero** [comment]
EStán en la carpeta de procesados

**Comment by bpesquero** [comment]
Verificado COM/00906 y ok
vertificado COM/03248 y ok

**Comment by bpesquero** [comment]
Creo que no quedaron bien asociados, no? O me estoy liando, me refiero a las tablas de toll y closing con la factura de venta y compra

**Comment by JMunoz17** [comment]
Asigné manualmente los números de factura correctos a los peajes, a los cierres y a las medidas. Lo acabo de revisar y no he visto nada raro, hay algo que no te cuadre?

**Comment by bpesquero** [comment]
No, creo que la primera vez que lo revise no estaba asociada y me quedé con la cosa de que había que hacerlo y luego se me pasó completamente hasta ayer. Si lo has revisado tú, perfecto
